packages/common/src/store/pages/profile/actions.ts

```diff
@@ -28,20 +24,6 @@ export type SetCurrentUserAction = {
   handle: string
 }
 
-export type UpdateProfileAction = {
```

Comment:
LOVE to see it

Reply:
<img width='40%' src='https://github.com/user-attachments/assets/38b209b7-9748-4d40-b066-ae4dfdb94999' />